stories/skipLink/skipLink.handlebars

```diff
@@ -1 +1 @@
-<a href="#main" class="skip-link {{class}}{{#if hidden}} visually-hidden{{/if}}">{{text}}</a>
+<a href="#main" class="skip-link">{{text}}</a>
```

Comment:
```suggestion
<a href="#main" class="skip-link"{{#if hidden}} visually-hidden{{/if}}>{{text}}</a>
```
Add this back in to be able to use the hidden true/false control in Storybook?

Reply:
Yeah, sorry, thought I had added that back in already! Still trying to work on changing the handlebars to what Hillel suggested.

Reply:
Yeah I'm not sure about this because it's possible for this component to be dropped in a web page and to work with just HTML and CSS. Adding/removing a class can only be done via JS, so I would prefer a different pattern I think.